backend/requirements/default.txt

```diff
@@ -36,6 +36,7 @@ psycopg2-binary==2.9.9
 pycryptodome==3.19.0
 pydantic==1.10.7
 PyGithub==1.58.2
+python-gitlab>=3.9.0
```

Comment:
prefer to pin to exact version or otherwise unexpected breakages can happen as libraries automatically upgrade